config/index.js

```diff
@@ -0,0 +1,4 @@
+import { Pool } from 'pg'
```

Comment:
@olamilekan000 , you can find a more descriptive name for this file, since its all about database.

Reply:
and also I will advise you to include the database config here so that you can specify a different database depending on the environment.